<?xml version="1.0" encoding="UTF-8"?>
<!-- SPDX-FileCopyrightText: Copyright (c) Kitware Inc. -->
<!-- SPDX-License-Identifier: BSD-3-Clause -->
<ui version="4.0">
 <class>QuickLaunchDialogExtended</class>
 <widget class="QDialog" name="QuickLaunchDialogExtended">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quick Launch</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLineEdit" name="Request">
     <property name="placeholderText">
      <string>Type to search</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QPushButton" name="CreateProxy">
     <property name="text">
      <string>SelectedElement</string>
     </property>
     <property name="toolTip">
      <string>Create the selected element</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="3" colspan="2">
    <widget class="QListView" name="AvailableProxies">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding"/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QToolButton" name="GoToHelp">
     <property name="text">
      <string>?</string>
     </property>
     <property name="toolTip">
      <string>Open the Help</string>
     </property>
     <property name="icon">
      <iconset resource="../pqComponents.qrc">
       <normaloff>:/pqWidgets/Icons/pqHelp.svg</normaloff>:/pqWidgets/Icons/pqHelp.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="ShortHelp">
     <property name="text">
      <string>Short help for selected element</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="RequirementIcon">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="Requirements">
     <property name="text">
      <string>Requirements (if unmet)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="DisabledLabel">
     <property name="text">
      <string>Unavailable:</string>
     </property>
     <property name="toolTip">
      <string>Select one to see its unmet requirements.</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QListView" name="DisabledProxies">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum"/>
     </property>
     <property name="toolTip">
      <string>Unavailable: select one to see its unmet requirements.</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="Instructions">
     <property name="text">
      <string>Press Enter to create, Shift+Enter to create and apply.
Esc to cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
